docs/sass/_normalize.scss

```diff
@@ -219,6 +220,8 @@ audio:not([controls]) {
 
 img {
   border-style: none;
+  max-width: 100%;
+  height: auto;
```

Comment:
Please add the rule in the main scss file, this is a vendored file

Reply:
@Keats Okay, Could you please tell me which is the main file? Is _docs.scss the main file?

Reply:
Put it in this file: https://github.com/getzola/zola/blob/master/docs/sass/_base.scss